### PR TITLE
ShiftUp.NIKKE version 0.0.6.178

### PR DESCRIPTION
**Pull request by solomoncyj**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: codeflow:open?pullrequest=https://github.com/microsoft/winget-pkgs/pull/110645&drop=dogfoodAlpha

### CODE REVIEW

File: manifests/s/ShiftUp/NIKKE/0.0.6.178/ShiftUp.NIKKE.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: ShiftUp.NIKKE
+PackageVersion: 0.0.6.178
+PackageLocale: en-US
+Publisher: Tencent Games
```
Comment (Trenly) [inline manifests/s/ShiftUp/NIKKE/0.0.6.178/ShiftUp.NIKKE.locale.en-US.yaml:7]:
```suggestion
Publisher: Level Infinite
```
This should match control panel

File: manifests/s/ShiftUp/NIKKE/0.0.6.178/ShiftUp.NIKKE.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: ShiftUp.NIKKE
```
Comment (Trenly) [inline manifests/s/ShiftUp/NIKKE/0.0.6.178/ShiftUp.NIKKE.installer.yaml:4]: Where does `ShiftUp` come from? The Installer URL and Publisher don't seem to match with `ShiftUp`

File: manifests/s/ShiftUp/NIKKE/0.0.6.178/ShiftUp.NIKKE.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: ShiftUp.NIKKE
+PackageVersion: 0.0.6.178
+PackageLocale: en-US
+Publisher: Tencent Games
+PackageName: 'Goddess of Victory: Nikke'
+License: proprietary
```
Comment (Trenly) [inline manifests/s/ShiftUp/NIKKE/0.0.6.178/ShiftUp.NIKKE.locale.en-US.yaml:9]:
```suggestion
License: Proprietary
```

**Review by Trenly — CHANGES_REQUESTED** [review]

File: manifests/s/ShiftUp/NIKKE/0.0.6.178/ShiftUp.NIKKE.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: ShiftUp.NIKKE
+PackageVersion: 0.0.6.178
+PackageLocale: en-US
+Publisher: Tencent Games
+PackageName: 'Goddess of Victory: Nikke'
```
Comment (Trenly) [inline manifests/s/ShiftUp/NIKKE/0.0.6.178/ShiftUp.NIKKE.locale.en-US.yaml:8]:
```suggestion
PackageName: NIKKE
```
This should match control panel

File: manifests/s/ShiftUp/NIKKE/0.0.6.178/ShiftUp.NIKKE.installer.yaml (hunk 2 above)
Comment (solomoncyj) [reply manifests/s/ShiftUp/NIKKE/0.0.6.178/ShiftUp.NIKKE.installer.yaml:4]: if you go to https://nikke-en.com/ and scroll all the way down, there it is

Comment (stephengillie) [reply manifests/s/ShiftUp/NIKKE/0.0.6.178/ShiftUp.NIKKE.installer.yaml:4]: > ©Proxima Beta Pte. Limited ©SHIFT UP CORP.

**Review by stephengillie — APPROVED** [review]

**Pull request merged**